hnn_core/gui/_viz_manager.py

```diff
@@ -273,11 +278,26 @@ def _dynamic_rerender(fig):
     fig.tight_layout()
 
 
-def _plot_on_axes(b, widgets_simulation, widgets_plot_type,
-                  target_simulations,
-                  spectrogram_colormap_selection, dipole_smooth,
-                  max_spectral_frequency, dipole_scaling, widgets, data,
-                  fig_idx, fig, ax, existing_plots):
+def create_plot_config(data, dipole_scaling, data_scaling, dipole_smooth,
+                       data_smooth, max_spectral_frequency,
+                       spectrogram_colormap_selection):
+
```

Comment:
```suggestion
""" Returns dict of plotting parameters from input widgets """

```

Reply:
Done

Reply:
I made this change in commit [e3ae1c4](https://github.com/jonescompneurolab/hnn-core/pull/730/commits/e3ae1c4d49d845c7b025126f9214dcce200598da), but it's not being reflected in the PR. Any ideas why ?